fusillade/clouddirectory.py

```diff
@@ -891,54 +893,63 @@ def get_policies(self, policy_paths: List[Dict[str, Any]], policy_type='IAMPolic
         )
 
         # retrieve the policies in a single request
-        operations = [
-            {
-                'GetObjectAttributes': {
-                    'ObjectReference': {'Selector': f'${policy_id}'},
-                    'SchemaFacet': {
-                        'SchemaArn': self.node_schema,
-                        'FacetName': 'POLICY'
-                    },
-                    'AttributeNames': ['policy_document']
-                }
-            }
+        operations = []
```

Comment:
looks like you might have a typo here with the extra `[]`

Reply:
that is the intended syntax, but it's not very pythonic. Will fix